cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/MaintenanceTimeWindowsTest.java

```diff
@@ -347,9 +445,9 @@ public boolean certRenewed() {
             }
         };
 
-        StatefulSet z = this.mockClient.apps().statefulSets().inNamespace(NAMESPACE).withName(KafkaResources.zookeeperStatefulSetName(NAME)).get();
+        StatefulSet z = this.mockClient.apps().statefulSets().inNamespace(NAMESPACE).withName(ZookeeperCluster.zookeeperClusterName(NAME)).get();
         z.getSpec().getTemplate().getMetadata().getAnnotations().put(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, "1");
-        this.mockClient.apps().statefulSets().inNamespace(NAMESPACE).withName(KafkaResources.zookeeperStatefulSetName(NAME)).patch(z);
+        this.mockClient.apps().statefulSets().inNamespace(NAMESPACE).withName(ZookeeperCluster.zookeeperClusterName(NAME)).patch(z);
```

Comment:
Out of interest, why did you change this back to using the `ZookeeperCluster` method?

Reply:
What I got from the Stanislav review (about ZK rolling update) is that we are providing `KafkaResources` just for the api which can be used by other people and that internally we can still use the model ones. Did I get it wrong?